### DISCUSSION TITLE
Update UploadFile (Image) information, like `alternativeText`

### DISCUSSION THREAD

**Discussion by willianjusten** [strapi/strapi#7359] [Legacy Topics]
Hey guys, how can I update the `alternativeText` from an image using GraphQL?

As I can see by playing with the administration, we basically send a `POST` to `/upload?id=${ID_IMAGE}` with a `form-data` with some of the information.

With that, I managed to make it work with this endpoint with Rest API. But how can I do the same using GraphQL? I wasn't able to find any mutations that allows me to edit this part, only to upload a new image =/

Am I missing something?

**Comment by lauriejim** [comment]
Hello! I just checked in the GraphQL schema and I haven't be able to see a mutation to update a file data.
I think it hasn't been developed when we released the Media Lib feature.
Can you please open a GitHub issue about that.
Thank you 👍
